lib/hooks/create/establishment.js

```diff
@@ -51,6 +51,16 @@ module.exports = settings => {
       case 'grant':
         return model.setStatus(withInspectorate.id);
 
+      case 'revoke':
+        return Promise.resolve()
+          .then(() => Profile.query().findById(changedBy))
+          .then(profile => {
+            if (!profile.asruUser || !profile.asruLicensing) {
```

Comment:
I would _prefer_ to go for a `true && true` here - rather than a `false || false` - also can a user ever be asruLicensing and not asruUser? If not then we could just do an `if (profile.asruLicensing)...`

Reply:
There was a time when we just removed `asruUser` and remembered `asruLicensing` if an LO was removed from ASRU, but that should no longer be the case. This is just for belt and braces.

Reply:
👍